tests/textcode/test_pdf.py

```diff
@@ -44,7 +44,7 @@ def test_get_text_lines(self):
 Extracts text from a pdf file.
 """
 import contextlib
-from StringIO import StringIO
+from io import StringIO
```

Comment:
DO NOT change this code... this is  an expectation test string. This is not code... even if it looks like it. See line 41   https://github.com/nexB/scancode-toolkit/pull/1608/files#diff-4fbe57782889037fd619e1df7509d369R41 ... this is a string!

Reply:
oh, but you have done here https://github.com/nexB/scancode-toolkit/commit/1e95b2c025acc82aec3e673012a04ee96f808602#diff-4fbe57782889037fd619e1df7509d369 ,  the test is still passed.

Reply:
Ok, I removing this, but i want to know why your commits passed all test.

Reply:
@Abhishek-Dev09 you want not to look at a single commit at a time but at the end game with all commits applied.

Reply:
And that https://github.com/nexB/scancode-toolkit/commit/1e95b2c025acc82aec3e673012a04ee96f808602#diff-4fbe57782889037fd619e1df7509d369L47 surely was a mistake of mine ;)